sass/edx/_variables.scss

```diff
@@ -645,22 +645,21 @@ $nav-tabs-justified-active-link-border-color: transparent !default;
 //
 // Define colors for form feedback states and, by default, alerts.
 
-$state-success-text:             $green !default;
-$state-success-bg:               $body-bg !default;
-$state-success-border:           $green !default;
+$state-success-text:               $black !default;
+//$state-success-bg:               #dff0d8 !default;
```

Comment:
Super-Nit: the rest of the page does not align the variables definitions that are commented out with those that aren't commented out.